test/stateless.test.js

```diff
@@ -58,8 +54,8 @@ test("stateless component with context support", async () => {
         </C.Provider>
     )
 
-    await asyncReactDOMRender(<ContextProvider />, testRoot)
-    expect(testRoot.querySelector("div").innerHTML.replace(/\n/, "")).toBe("context: hello world")
+    const { container } = render(<ContextProvider />)
+    expect(container.querySelector("div").innerHTML.replace(/\n/, "")).toBe("context: hello world")
```

Comment:
Shouldn't this work as well? There are more places like that in here. It's more readable imo.
```suggestion
    expect(getByText("context: hello world")).toBeTruthy()
```

Reply:
Thanks, I will investigate this later today.

Reply:
I do not think that this is the good idea because getByText will throw an error if the element will not be found, so the assertion will be redundant. I simplified assertions a bit.

Reply:
Fair enough, doesn't matter in simple cases like that. However, in apps, I often do just `getByText` without expect because assertion is nothing else than throwing an error :) The output in case of failure does not differ that much.

Reply:
OK, thanks. I prefer assertions via `expect` because with them it becomes more clear what expectations we have in tests.